### PR TITLE
Feature/orca 301 Create input/output/config schema for get_current_archive_list

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Please write a sentence or two summarizing the proposed change.

Addresses [ORCA-301: Create input/output/config schema for get_current_archive_list](https://bugs.earthdata.nasa.gov/browse/ORCA-301)

## Changes

* Added schemas for get_current_archive_list
* Added basic files for lambda

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

It has not.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
    - [x] Manual testing/integration testing passes (if forked)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

**Review by rizbih — APPROVED** [review]
FYI output of this lambda will be input of perform_orca_reconcile lambda which is the 2nd step of the step function workflow.

File: tasks/get_current_archive_list/bin/run_tests.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# Currently just running unit tests until we fix/support large tests
+coverage run --source get_current_archive_list -m pytest
+let return_code=$?
+check_rc $return_code "ERROR: Unit tests encountered failures."
+
+# Unit tests expected to cover minimum of 80%.
+coverage report --fail-under=80
+let return_code=$?
+check_rc $return_code "ERROR: Unit tests coverage is less than 80%"
+
```
Comment (bhazuka) [inline tasks/get_current_archive_list/bin/run_tests.sh:81]: May want to add additional tests like linting, code smell, formatting, etc. like we have in https://github.com/nasa/cumulus-orca/blob/master/shared_libraries/bin/run_tests.sh

**Review by bhazuka — CHANGES_REQUESTED** [review]
You may want to copy over and modify the api creation script from https://github.com/nasa/cumulus-orca/blob/master/shared_libraries/bin/build_api.sh

**Review by bhazuka — APPROVED** [review]
Looks good

**Pull request merged**